buildkite/build_project.sh

```diff
@@ -206,10 +210,15 @@ case "$REPO_FULL_NAME" in
         ;;
 
     ldc-developers/ldc)
-        git submodule update --init
+        cd runtime && git clone --depth 1 https://github.com/ldc-developers/druntime
+        git clone --depth 1 https://github.com/ldc-developers/phobos
+        cd .. && mkdir bootstrap && cd bootstrap
```

Comment:
Wait, so you're using the latest `ldc` heads for the 2 submodules? That's not really master and might be problematic at some point (because untested in this particular combination).